### ISSUE TITLE
Document pros/cons vs. competition

### ISSUE THREAD

**Issue by davidbgk** [vvo/lazyload#72]
It might be interesting to add a page/section about differences with other similar scripts like [[be]Lazy](http://dinbror.dk/blazy/) for instance. Thoughts?

**Comment by vvo** [comment]
Yes that's a good idea, I am taking PR on this, I am not actively using lazyload even if it works well.

blazy was done after my lazyload so I am not sure what are the differences. Maybe in the number of edge cases solved.

**Comment by davidbgk** [comment]
http://developer.telerik.com/featured/lazy-loading-images-on-the-web/ might be a good resource to start with.
